src/stores/settingsStore.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @copyright Copyright (c) 2020 Joas Schilling <coding@schilljs.com>
+ *
+ * @author Maksim Sukharev <antreesy.web@gmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { defineStore } from 'pinia'
+
+import { getCapabilities } from '@nextcloud/capabilities'
+import { loadState } from '@nextcloud/initial-state'
+
+import { PRIVACY } from '../constants.js'
+import { setReadStatusPrivacy, setTypingStatusPrivacy } from '../services/settingsService.js'
+
+export const useSettingsStore = defineStore('settingsStore', {
+	state: () => ({
```

Comment:
**nitpick:** store key format

I'm not sure we need should add `*Store` prefix to all stores' keys. This key is used only internally in Pinia and in the store devtools. So with this prefix, we will have `*Store` for every item in the list of stores in the store dev tools.

This is also not how stores are named in the [docs](https://pinia.vuejs.org/core-concepts/#defining-a-store), and not how pinia stores are called in other places there Pinia is used in nextcloud, for example, in files:
https://github.com/nextcloud/server/blob/686ad69133bb9359f9386080aface284c8639398/apps/files/src/store/viewConfig.ts#L33-L37

Reply:
PS I mean only the key passed to the `defineStore`, not the composable variable name.